__tests__/src/components/GalleryView.test.js

```diff
@@ -19,28 +18,27 @@ function createWrapper(props) {
 
 describe('GalleryView', () => {
   let setCanvas;
-  let wrapper;
   beforeEach(() => {
     setCanvas = jest.fn();
-    wrapper = createWrapper({ setCanvas });
   });
   it('renders the component', () => {
-    expect(wrapper.find(Paper).length).toBe(1);
-    expect(wrapper.find(Paper).prop('component')).toEqual('section');
+    createWrapper({ setCanvas });
+    const buttons = screen.queryAllByRole('button');
+    expect(buttons[0].closest('section')).toBeInTheDocument(); // eslint-disable-line testing-library/no-node-access
```

Comment:
This seems a little roundabout -- just means if any ancestor is a `section` ? If the buttons/GalleryViewThumbnails are rendering, then can we just assume the component is rendering? Like maybe the first two `it` blocks end up being the same. Not sure.